proto/reporting/rpc/front_desk_reporting_svc.proto

```diff
@@ -45,32 +46,44 @@ message FrontDeskReportingSvcPropertyOpsdateRequest {
 	holms.types.tenancy_config.indicators.PropertyIndicator property = 2;
 }
 
+message FrontDeskBatchReportingSvcRequest {
+	repeated holms.types.tenancy_config.indicators.PropertyIndicator properties = 1;
```

Comment:
OK, sure.

You can do it like this, or the simpler filter implementation I suggested before. Your call.

Putting the property out here but the other parameters inside the manifest itself seems a little inconsistent, but I can live with it.